_core/smart/neon-node.md

```diff
@@ -56,7 +44,7 @@ You're all set to run a node that connects to the Neon network.
 Back in the command line, run:
 
 ```bash
-cargo testnet ./testnet/follower-conf.toml
+stacks-node neon
 ```
 
 The first time you run this, you'll see some logs indicating that the Rust code is being compiled. Once that's done, you should see some logs that look something like the this:
```

Comment:
I actually now only see one INFO:
```
INFO [1587676136.184] [src/chainstate/stacks/index/marf.rs:732] First-ever block 0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206
```

Reply:
Because the POST and endpoint isn't displayed now, how would people know what endpoint to call in the RPC API? The port is unknown to them, right?

Reply:
Also calling out that there will likely be errors visible in here and it could take a while until a new log shows. might be known to the reader, but just a nit on my side.